### PR TITLE
Update recommended block list to explain not blocking 1903 files

### PR DESCRIPTION
**Pull request by isbrahm**
msxml3.dll, msxml6.dll, and jscript9.dll do not have to be blocked if using 1903, as the previous issue was fixed in this release

### CODE REVIEW

**Comment by isbrahm** [comment]
@Dansimp could you review and merge this?

**Comment by illfated** [comment]
May I suggest a minor readability improvement?
```diff
-  <!-- pick correct version of msxml3.dll, msxml6.dll, and jscript9.dll based on release you are supporting -->
+  <!-- pick the correct version of msxml3.dll, msxml6.dll, and jscript9.dll based on the release you are supporting -->
```
(add a couple of definite articles "the" to make it a more natural sentence in English)

**Comment by illfated** [comment]
Thanks. Looks good now.

File: windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md [hunk 1]
```diff
@@ -893,7 +892,7 @@ Pick the correct version of each .dll for the Windows release you plan to suppor
   <FileRuleRef RuleID="ID_DENY_WMIC"/>
   <FileRuleRef RuleID="ID_DENY_MWFC" /> 
   <FileRuleRef RuleID="ID_DENY_WFC" /> 
-  <!-- Uncomment the relevant line(s) below if you have uncommented them in the rule definitions above.
+  <!-- uncomment the relevant line(s) below if you have uncommented them in the rule definitions above
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md]: By the way, is this change actually necessary? Is there some kind of "unwritten rule" that comment lines should be without capitalization and full stops (periods)?

Comment (isbrahm) [reply windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md]: Not that I know of. I changed this for consistency. I could also edit the other changes to match this one instead (i.e. use capitalization and full stops)

Comment (illfated) [reply windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md]: That sounds good to me. I support your suggestion. 👍

**Comment by denisebmsft** [comment]
Thank you @isbrahm and @illfated
